scraper/middlewares.py

```diff
@@ -0,0 +1,21 @@
+# very basic subdomain blocking middleware to intercept requests and drop them if the url is for
```

Comment:
added by me, middleware concept is supported by the scrapy architecture though, and this middleware is enabled via the settings.py file